front/components/assistant_builder/TeamSharingSection.tsx

```diff
@@ -112,15 +111,16 @@ export function TeamSharingSection({
             private version.
           </div>
           <div>
-            <Tooltip label="Coming soon: you can create a duplicate to tweak your own, private version">
+            <Link
+              href={`/w/${owner.sId}/builder/assistants/${agentConfigurationId}?flow=personal_assistants&copy=true`}
```

Comment:
This is quite disatisfying that the URL preserve the old agentConfigurationId

Let's instead use `assitsants/new?duplicate={agentConfigurationId}`

Reply:
I worry it will lead to bad bugs in the future. Pre-filling a `new` page looks much safe